omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -321,4 +345,27 @@ private Boolean isNewUser(User user) {
 		return user == null ? true : user.getUserId() == null;
 	}
 	
+	/**
+	 * @return true if email is valid or false otherwise
+	 * @param email
+	 */
+	private boolean isEmailValid(String email) {
+		return StringUtils.hasLength(email) && EmailValidator.getInstance().isValid(email);
+	}
+	
+	/**
+	 * @return an email field
+	 * @param emailFieldName
+	 * @param user
+	 */
+	private Field getEmailField(String emailFieldName, User user) {
```

Comment:
Do you need to pass these as parameters? Do you plan to reuse this method for other parameters?

Reply:
Why not do something like this: https://github.com/openmrs/openmrs-sdk/blob/5.5.0/sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java#L805-L807
And then set the email field.

Reply:
Here is a second example: https://github.com/openmrs/openmrs-module-xforms/blob/4.3.14/omod/src/main/java/org/openmrs/module/xforms/page/controller/PatientRegPageController.java#L102

https://github.com/openmrs/openmrs-module-xforms/blob/4.3.14/omod/src/main/webapp/patientReg.jsp#L123

Reply:
With this approach, i have used ` User.class.getDeclared...` I hope it is not a bad practice